frontend/src/main/pages/SectionSearches/SectionSearchesIndexPage.js

```diff
@@ -34,6 +36,44 @@ export default function SectionSearchesIndexPage() {
     mutation.mutate(query);
   }
 
+  const {
+    data: _subjects,
+    error: _error,
+    status: _status,
+  } = useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    ["/api/UCSBSubjects/all"],
+    // Stryker disable next-line all : stryker changing "GET" to "" does nothing; "get" is the default method
+    { method: "GET", url: "/api/UCSBSubjects/all" },
+    [],
+  );
+
+  const objectToAxiosParamsSubjects = () => ({
+    url: "/api/UCSBSubjects/load",
+    method: "POST",
+  });
+
+  const onSuccessSubjects = (subjects) => {
+    toast(`Number of Subjects Loaded : ${subjects.length}`);
+  };
+
+  const mutationSubjects = useBackendMutation(
+    objectToAxiosParamsSubjects,
+    { onSuccess: onSuccessSubjects },
+    // Stryker disable next-line all : hard to set up test for caching
+    ["/api/UCSBSubjects/all"],
+  );
+
+  //not sure how to mutation test for empty dependency list
+  // Stryker disable all
+  useEffect(() => {
+    mutationSubjects.mutate();
+    //not sure how to get around eslint for an empty dependency
+    //list since I have to make it empty to only run on mount
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  // Stryker enable all
```

Comment:
Please change to Stryker restore all, since enable all does not work

Reply:
done